end_to_end/tpu/llama3/8b/2_test_llama3_8b.sh

```diff
@@ -0,0 +1,63 @@
+#!/bin/bash
+
+# This file is both an integration test that runs once a day on a v4-16 and documentation for how to get started with Llama3-8b. 
+# Please make sure you have run end_to_end/tpu/llama3/8b/1_test_llama3_8b.sh before running commands from this file. 
+
+# The flow of this file is as follows:
+# 1. Run decoding, finetuning of Llama3-8B with the converted checkpoint obtained from end_to_end/tpu/llama3/8b/1_test_llama3_8b.sh. Also, run pretraining of Llama3-8B
+# 2. Convert the scanned checkpoint from step 1 into unscanned checkpoint format and run more efficient decoding.
```

Comment:
since the conversion into unscanned checkpoint has been moved to script #1, let's change this to `Run more efficient decoding with the unscanned checkpoint obtained from end_to_end/tpu/llama3/8b/1_test_llama3_8b.sh`

Reply:
Done. Thanks